src/app/pages/tutorial/tutorial.page.ts

```diff
@@ -9,9 +9,21 @@ import { OnboardingService } from '../../services/onboarding/onboarding.service'
   styleUrls: ['./tutorial.page.scss'],
 })
 export class TutorialPage implements OnInit {
+  slideOptsOne = {
+    initialSlide: 0,
+    slidesPerView: 1,
+  };
+
   constructor(private readonly onboardingService: OnboardingService) {}
 
   ngOnInit() {
     this.onboardingService.onboard();
   }
+
+  // tslint:disable-next-line: prefer-function-over-method prefer-method-signature no-inferred-empty-object-type
+  next(slide: { slideTo: (arg0: number) => {} }, index: number) {
```

Comment:
Remove this method by using the `slides` template variable directly in the template.